skt/__init__.py

```diff
@@ -405,11 +405,19 @@ def merge_patchwork_patch(self, uri):
                           patchinfo.get("name").replace(',', ';')))
 
     def merge_patch_file(self, path):
+        if not os.path.isfile(path):
+            raise Exception("Patch %s not found" % os.path.basename(path))
+        abspath = os.path.abspath(path)
         try:
-            self.git_cmd("am", path)
-        except subprocess.CalledProcessError:
+            subprocess.check_output(["git", "am", abspath], cwd=self.wdir)
+        except subprocess.CalledProcessError, stdout:
```

Comment:
This works with Python2 (which we currently support), but the syntax is deprecated in Python3. As we'd eventually like to support both in the future, it would be easier to use `except subprocess.CalledProcessError as <identifier_name>:` syntax (which is compatible) instead of `,`. Using `stdout` as identifier for exception is also confusing, usually `exc` is being used. Can you change it?

Reply:
That makes sense!